index.js

```diff
@@ -27,29 +28,29 @@ function copypaste(from, to, ...props) {
  */
 function prepare(props) {
   const clean = rip(props,
-    'translate', 'scale', 'rotate', 'skewX', 'skewY',
-    'fontFamily', 'fontSize', 'fontWeight', 'fontStyle'
-  );
+        'translate', 'scale', 'rotate', 'skewX', 'skewY',
+        'fontFamily', 'fontSize', 'fontWeight', 'fontStyle'
+    );
 
   const transform = [];
   const style = {};
 
-  //
-  // Correctly apply the transformation properties.
-  //
+    //
+    // Correctly apply the transformation properties.
+    //
   if ('translate' in props) transform.push(`translate(${props.translate})`);
   if ('scale' in props) transform.push(`scale(${props.scale})`);
   if ('rotate' in props) transform.push(`rotate(${props.rotate})`);
   if ('skewX' in props) transform.push(`skewX(${props.skewX})`);
   if ('skewY' in props) transform.push(`skewY(${props.skewY})`);
   if (transform.length) clean.transform = transform.join(' ');
 
-  //
-  // This is the nasty part where we depend on React internals to work as
-  // intended. If we add an empty object as style, it shouldn't render a `style`
-  // attribute. So we can safely conditionally add things to our `style` object
-  // and re-introduce it to our `clean` object
-  //
+    //
```

Comment:
Hmm, not sure how that happened, didn't touch any indentation manually, must've been some formatting tool applied by mistake. Reverted.